sdk/spring/spring-cloud-azure-starter-monitor-test/pom.xml

```diff
@@ -23,6 +23,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <spotbugs.skip>true</spotbugs.skip>
     <codesnippet.skip>true</codesnippet.skip>
+    <opentelemetry.version>1.28.0</opentelemetry.version> <!-- Only required with Spring Boot 3 to override the OTel version used in the Spring Boot BOM-->
```

Comment:
oh, I didn't realize this would override the `opentelemetry.version` used in the spring bom? nice

Reply:
yes, it overrides the OTel version of the Spring BOM